version.php

```diff
@@ -26,7 +26,7 @@
 defined('MOODLE_INTERNAL') || die();
 
 $plugin->version = 2022082500;
-$plugin->release = 2022082500;
+$plugin->release = "2022082500 for Moodle 3.5+";
```

Comment:
Do we need the words? Not saying we shouldn't just we should be consistent so if there is a good reason then lets do it across the board

Reply:
We don't, but the release field def says "String describing...". I just thought it was appropriate. Otherwise it's just a copy of version, no added value.

Reply:
We generally just make it the same purely to make it quick and simple. We have not ever bothered to do a version like 'RC1' or 'Release 1.3.5' because its just more work to maintain for not much value